Documentation/production-users.md

```diff
@@ -130,3 +130,17 @@ PD(Placement Driver) is the central controller in the TiDB cluster. It saves the
 - *Backups*: Periodic sync to backup server
 
 In Tencent games, we use Docker and Kubernetes to deploy and run our applications, and use etcd to save meta data for service discovery, Kubernetes, etc.
+
+## Hyper.sh
+
+- *Application*: Kubernetes, distributed locks, etc.
+- *Launched*: April 2016
+- *Cluster Size*: 1 cluster of 3 members
+- *Order of Data Size*: 10s of MB
+- *Operator*: Hyper.sh
+- *Environment*: Baremetal
+- *Backups*: None, all data can be recreated if necessary.
+
+[hypernetes](https://github.com/hyperhq/hypernetes)
```

Comment:
can you put a few sentences to explain your use case here? not just like these 3 key words? Thanks!